.github/workflows/build.yml

```
@@ -0,0 +1,185 @@
+name: build
+
+on:
+  push:
+    branches: [ master, '[1-9][0-9].[0-9]' ]
+  pull_request:
+    branches: [ master, '[1-9][0-9].[0-9]' ]
+
+jobs:
+  build:
+
```

```suggestion
    if: "!contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.head_commit.message, '[skip travis]')"
```
Skip the build commit message markers